pkg/metrics/chaos-controller-manager.go

```diff
@@ -53,7 +54,10 @@ func NewChaosControllerManagerMetricsCollector(manager ctrl.Manager, registerer
 		store = manager.GetCache()
 	}
 
+	logger, _ := z.NewDefaultZapLogger()
+
```

Comment:
please do not create another logger, acquire the logger as a parameter in the constructor.